src/devices/bituo_technik.ts

```diff
@@ -1,25 +1,144 @@
-import {Definition} from '../lib/types';
 import * as exposes from '../lib/exposes';
 import * as reporting from '../lib/reporting';
-import fz from '../converters/fromZigbee';
+import * as utils from '../lib/utils';
+import {
+    precisionRound,
+    postfixWithEndpointName, calibrateAndPrecisionRoundOptions,
+} from '../lib/utils';
+import {KeyValueAny, Fz, Definition} from '../lib/types';
 const e = exposes.presets;
 
+const fzLocal = {
+    seMetering: {
```

Comment:
Please extend the existing converters in fromZigbee.ts to prevent code duplication.

Reply:
Yes, I understand this, but we found that our converter is different from the one in fromZigbee. If we modify it in fromZigbee, it will affect the use of other devices, so I chose to add my own converter in the localfz way.

Reply:
>  but we found that our converter is different from the one in fromZigbee 

It looks generic, so I then we can add it.

Reply:
OK, I will resubmit the request, thank you